mailchimp3/baseapi.py

```diff
@@ -42,16 +42,28 @@ def _iterate(self, url, **kwargs):
         kwargs['count'] = integer
         kwargs['offset'] = integer
         """
+        #fields as a kwarg ought to be a string with comma-separated substring
+        #values to pass along to self._mc_client._get(). it also ought to
+        #contain 'total_items' in all cases, so that is enforced here
+        if kwargs.get('fields') is None:
+            kwargs['fields'] = 'total_items'
```

Comment:
Change this to `pass` as doing this would return only the count of records instead of the records. When fields is not provided, the API assumes all fields should be sent